src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -17,7 +69,11 @@ export default class MutationInformationContainer extends React.Component<IMutat
                 visible: "excluded"
             },
             tumors: {
-                name: "Tumors"
+                name: "Tumors",
+                formatter: TumorColumnFormatter.makeRenderFunction(this.props.sampleOrder, this.props.sampleColors, this.props.sampleLabels,
```

Comment:
But it's not a renderFunction, it's making a renderFunction (this goes back to the 'get' naming debate from the other day 😛 )